qa/tasks/ceph_manager.py

```diff
@@ -110,11 +110,21 @@ def mount_osd_data(ctx, remote, cluster, osd):
             )
 
 
+def log_exc(func):
+    @wraps(func)
+    def wrapper(self):
+        try:
+            return func(self)
+        except:
+            self.log(traceback.format_exc())
+            raise
+    return wrapper
+
+
 class PoolType:
     REPLICATED = 1
     ERASURE_CODED = 3
 
-
```

Comment:
As this is toplevel,  I'd leave this blank linke to stay pep8 compatible.

Reply:
ah. sorry. fixed